### PR TITLE
Add automated update version and create pull request bot

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
Hi @pragnagopa 

This PR is for the automation of the version up and PR.  After merging this PR, I'll create a new one that merge dev to release. 

This is a bot. However, it won't automatically trigged. It will trigged manually. My plan is, kick this pipeline from Azure DevOps Linux pipeline with the rest api. 

I tested this yaml file on my repository, however, need to test it in this repo. 

image

image

### PR information

- [-] The title of the PR is clear and informative.
- [-] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).

### CODE REVIEW

File: .github/workflows/autopr.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: dev
+    - id: createBranch
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
```
Comment (pragnagopa) [inline .github/workflows/autopr.yaml:24]: We have functions git hub user : https://github.com/azfuncgh :)

Lets use same user across repos. Reach out to  @yojagad  if needed , on how to use this user. azfuncgh is used in host and webjobs sdk repos for automated PRs

**Comment by pragnagopa** [comment]
@yojagad - please review. This is part of automating Linux release pipelines.

File: .github/workflows/autopr.yaml (hunk 1 above)
Comment (TsuyoshiUshio) [reply .github/workflows/autopr.yaml:24]: Oh, I didn't know that! Hi @yojagad I'd like to know the e-mail account of azfuncgh !

Comment (pragnagopa) [reply .github/workflows/autopr.yaml:24]: Fixed Yogesh's alias in above comment.

Comment (yojagad) [reply .github/workflows/autopr.yaml:24]: It's azfuncgh@github.com . If you need PAT for that, please ping @fabiocav for the same :)

File: .github/workflows/autopr.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      id: createPullRequest
+      uses: peter-evans/create-pull-request@v3
+      with:
+        title: Update to verison ${{ github.event.inputs.targetVersion }}
+        body: |
+          - Update ${{ github.event.inputs.majorVesion}} to ${{ github.event.inputs.targetVersion }}
+        branch: action/release.${{ github.event.inputs.targetVersion }}
+        labels: |
+          automated pr
+        reviewers: tsuyoshiushio
```
Comment (yojagad) [inline .github/workflows/autopr.yaml:39]: >tsuyoshiushio [](start = 19, length = 13)

Add other reviewers?

File: .github/workflows/autopr.yaml (hunk 1 above)
Comment (pragnagopa) [reply .github/workflows/autopr.yaml:24]: thanks. Will follow up with @fabiocav offline.

File: .github/workflows/autopr.yaml (hunk 2 above)
Comment (TsuyoshiUshio) [reply .github/workflows/autopr.yaml:39]: Absolutely! This pr is for testing. ( we need to test it first). So that I just added me at first.

**Review by yojagad — APPROVED** [review]
:shipit:

File: .github/workflows/autopr.yaml (hunk 2 above)
Comment (yojagad) [reply .github/workflows/autopr.yaml:39]: Thanks for clarifying ! if this is just for testing, might be worth checking this in, testing this out and then make updates on this later if we're missing anything.

---
In reply to: [599829115](https://github.com/Azure/azure-functions-docker/pull/410#discussion_r599829115) [](ancestors = 599829115)

**Review by pragnagopa — APPROVED** [review]

**Pull request merged**